actions.js

```diff
@@ -3039,6 +3040,12 @@ ActionManager.prototype.afterActionApplied = function(/*msg*/) {
     // a custom block)
 
     active.onSetActive();
+    if (this.emitActions) {
+        window.parent.postMessage({
```

Comment:
This should be updated to use the equivalent of `event.source` as has been updated in `initializeEmbeddedAPI`